scripts/gas-refund-program/staking/abstract-stakes-tracker.ts

```diff
@@ -1,15 +1,37 @@
 import BigNumber from 'bignumber.js';
+import { assert } from 'ts-essentials';
 
 export class AbstractStakesTracker {
   startBlock: number;
   endBlock: number;
+  startTimestamp: number;
+  endTimestamp: number;
 
-  setBlockBoundary(startBlock: number, endBlock: number) {
+  setBlockBoundary({
+    startBlock,
+    endBlock,
+    startTimestamp,
+    endTimestamp,
+  }: {
+    startBlock: number;
+    endBlock: number;
+    startTimestamp: number;
```

Comment:
Second thought: I'm thinking better do the resolution of `(start|end)Timestamp` out of `(start|end)Block` from here. It'd smoothen test readability as we'd just have to declare purely block based boundary 🤔

Reply:
wiser to not address today though

Reply:
today I'm crazy enough to address [it](https://github.com/paraswap/paraswap-volume-tracker/pull/60)